src/styles/annotator/notebook.scss

```diff
@@ -0,0 +1,21 @@
+@use '../variables' as var;
+@use '../mixins/molecules';
+
+.notebook-outer {
+  // TODO: CSS namespace conflicts?
+  box-sizing: border-box;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  padding: var.$layout-space;
+  // Leave explicitly the right amount of room for closed-sidebar affordances
+  padding-right: var.$annotator-toolbar-width + 5px;
+
+  .notebook-inner {
+    box-sizing: border-box;
```

Comment:
I'm moving fast here, but in its present structure, yes, `box-sizing` does appear to be necessary on both of these elements. Remove it here and the box fits correct, width-wise, but is too tall for the screen. I didn't take the time to deeply debug.